files/domjudge.desktop

```diff
@@ -1,15 +1,6 @@
 [Desktop Entry]
-BinaryPattern=
-Protocols=
-Comment=DOMjudge
-Name=DOMjudge
-Exec=gnome-www-browser https://domjudge/
-Icon=/usr/share/apps/domjudge/DOMjudgelogo.png
-TerminalOptions=
-Path=
+Version=1.0
 Type=Application
-Terminal=0
-MapNotify=false
-Encoding=UTF-8
-X-Desktop-File-Install-Version=0.1
-Categories=Contest
+Name=DOMjudge
+Exec=/usr/bin/firefox https://domjudge/
```

Comment:
Did the original one not work? I seem to recall we just didn't install it at all, and then pushed the firefox one kinda last minute. But the original looks like it'd be better: e.g. if we switched to chrome or something it'd still work, plus it had the domjudge logo as the icon.

Reply:
i think the original one required a logo from the ccs == domjudge block that was commented out.

Reply:
as i have fixed that block in a different PR,  i reverted the content changes to that file